editor/slices/loadSlice.tsx

```diff
@@ -1,53 +1,68 @@
 /* eslint-disable no-param-reassign */
-import { createSlice } from "@reduxjs/toolkit";
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import {
+  loadState,
+  textureType,
+  loadType,
+  DancersType,
+} from "../types/loadSlice";
+import {
+  ControlMapType,
+  ControlRecordType,
+  posRecordType,
+  lightPresetsType,
+  posPresetsType,
+} from "../types/globalSlice";
+import { RootState, AppDispatch } from "../store/index";
 
+const initialState: loadState = {
```

Comment:
Please add the comments back.

Reply:
![image](https://user-images.githubusercontent.com/79928691/151301333-3088ea54-83f6-4b1e-80b3-c6cc5710e9a8.png)
I put all these commands to /types/loadSlice.d.ts